php/class-fieldmanager-group.php

```diff
@@ -350,6 +350,11 @@ public function presave( $values, $current_values = array() ) {
 			}
 		}
 
+		// Handle empty groups.
+		if ( empty( $values ) ) {
+			$values = array();
```

Comment:
L343 above also assumes `$values` is an array. Is there ever a case where `$values` could be a legal non-empty non-array? If not, should an exception be thrown before L343 if one is passed?